docs/send-data/kubernetes/troubleshoot-collection.md

```diff
@@ -395,6 +490,268 @@ kube-prometheus-stack:
 
 where `metadata.name` is the value from Argo Application manifest.
 
+### Check metrics content
+
+You can print metrics on stdout of metrics collector and metrics metadata, and validate if they are correct. It may happen that metrics are ingested, but with different metadata than you expect them.
```

Comment:
```suggestion
You can print metrics on stdout of metrics collector and metrics metadata, and validate if they are correct. It may happen that metrics are ingested, but with different metadata than you expect.
```